run_ocsci.py

```diff
@@ -115,6 +116,7 @@ def main():
         '-p', 'ocsci.pytest_customization.marks',
         '-p', 'ocsci.pytest_customization.ocsci_logging',
     ])
+    add_path_to_env_path(os.path.join(HERE, 'bin'))
```

Comment:
@mbukatov what do you think about usage of [`bin_dir`](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/default_config.yaml#L38) configuration option here?

Reply:
I don't think it's necessary (as i'm not aware of any use case which would require this). I would suggest to leave this as it is now, and if it turns out I'm wrong we can add this later.